### DISCUSSION TITLE
Add MacPorts installation instructions?

### DISCUSSION THREAD

**Discussion by breun** [microsoft/openjdk#576] [Q&A] [answered]
I maintain various OpenJDK ports for [MacPorts](https://www.macports.org), including ports for the [Microsoft builds of OpenJDK](https://ports.macports.org/search/?q=openjdk+microsoft&name=on). 

Would you welcome the addition of a section on the [Install the Microsoft Build of OpenJDK](https://learn.microsoft.com/nl-nl/java/openjdk/install) about this installation option? If so, how could this be contributed exactly? Is there a public repository for this documentation?

**Comment by karianna** [comment]
@brunoborges over to you for guidance

**Comment by karianna** [comment]
This is odd, I thought I saw @brunoborges answer here but that has dissappeared?

**Comment by breun** [comment]
I don't know. 🤷‍♂️ I'd still be happy to contribute the installation instructions for MacPorts users if you can tell me how to go about this.

**Comment by karianna** [comment]
@brunoborges for when he comes back from OOO

**Comment by brunoborges** [comment, marked as answer]
Hi @breun, I appreciate your interest in contributing with MacPorts installation guide into our documentation.

Due to limited people in our documentation team, we won't be able to accept this as it is not an environment we officially support, and having those instructions may lead to confusion and documentation maintenance overhead.

**Reply by breun** [reply]
I think the maintenance overhead would be pretty minimal, because it would only mention one install commands for each provided Java version (e.g. `sudo port install openjdk17-microsoft`, `sudo port install openjdk21-microsoft`) for MacPorts users. And of course it could mention that this is not an installation method that is officially supported by Microsoft. But if you don't want to refer to MacPorts in your docs, then fair enough of course.

**Reply by brunoborges** [reply]
I do appreciate your help, but indeed this is something that, because we won't officially support, we can't document.

If there is anything else we can do to help, let us know.

**Reply by breun** [reply]
There are many projects that I was able to contribute MacPorts installation instructions to, even though they don’t officially support the installation method themselves, e.g. Spring Boot, etc. But I guess Microsoft is a different game. 🤷‍♂️
